tests/integration/test_logcollector/test_macos/test_macos_log_process.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import platform
+import signal
+
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import search_process, control_service
+from wazuh_testing.tools.utils import retry
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+macos_sierra = True if str(platform.mac_ver()[0]).startswith('10.12') else False
+
+# Marks
+
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=0)]
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_macos_format_basic.yaml')
+
+configurations = load_wazuh_configurations(configurations_path, __name__)
+
+
+# fixtures
+@pytest.fixture(scope="module", params=configurations, ids=[''])
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@retry(AssertionError, attempts=5, delay=2, delay_multiplier=1)
+def check_process_status(process_list, running=True, stage=''):
+    """Assert that some process are running or not.
+
+        This will check a list of process and asserts that all of them are running or not based on the arguments.
+
+        Args:
+            process_list (string list): the list of process to check
+            running (boolean): if the process are expected to be running or not
+            stage (string): in case of failure this string is append at the end of the error message and indicated
+                in which moment was the error produced (for example: after agent restart)
+
+        Raises:
+            AssertionError: if the condition is not meet
+    """
+    expected_process = 1 if running else 0
+    is_running_msg = 'is not running' if running else 'is running'
+    for process in process_list:
+        log_processes = search_process(process)
+        assert len(log_processes) == expected_process, f'Process {process} {is_running_msg} {stage}.'
+
+
+def test_independent_log_process(get_configuration, configure_environment, restart_logcollector):
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over. That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
The reasons for this change are:

- Faster testing
- More encapsulated tests